new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -565,7 +667,7 @@ private static JsonElement convertToJson(Object obj) {
    */
   public static boolean deleteDomainCustomResource(String domainUid, String namespace) {
 
-    KubernetesApiResponse<Domain> response = crdClient.delete(namespace, domainUid);
+    KubernetesApiResponse<Domain> response = crdClient.delete(namespace, domainUid, deleteOptions);
```

Comment:
finalizer can also prevent this from working - also needs to be run in a thread with a timeout